pkgs/os-specific/bsd/freebsd/pkgs/getty.nix

```diff
@@ -0,0 +1,21 @@
+{
+  mkDerivation,
+  login,
+  useWrappedLogin ? true,
+}:
+mkDerivation {
+  path = "libexec/getty";
+
+  postPatch = ''
+    sed -E -i -e "s|/usr/bin/login|${
+      if useWrappedLogin then "/run/wrappers/bin/login" else "${login}/bin/login"
+    }|g" $BSDSRCDIR/libexec/getty/*.h
+  '';
```

Comment:
Maybe this wrapping stuff should be left for later? Not really sure what it is yet.

Reply:
What's wrong with it? we are in fact doing suid binary wrappers on nixbsd and they work fine. Would you rather it default to off? What's the practical use of running a non-suid login binary?

Reply:
I just meant that `/run/wrappers` isn't a Nixpkgs thing, so it's arguably a minor layer violation I think?

Reply:
I changed the parameter to be the wrappedLogin path itself, null if unused, as we discussed on matrix. You should go complain to the other package maintainers for their packages that hardcode wrapper paths!